src/main/java/anichan/parser/BookmarkParser.java

```diff
@@ -80,10 +82,11 @@ private void parameterParser(String paramGiven) throws AniException {
             break;
         case LIST_PARAM:
```

Comment:
Perhaps you could standardize params naming to be `PARAMETER_NAME`?

Reply:
I understand the rationale with PARAMETER_NAME but I am consistent with the rest of our teammates,